dev-libs/gtest/patches/gtest-1.14.0.patchset

```diff
@@ -0,0 +1,43 @@
+From 0c4371e13b0cd8c85b0009c6024d1843d28da385 Mon Sep 17 00:00:00 2001
+From: "Mika T. Lindqvist" <postmaster@raasu.org>
```

Comment:
Keep credits to previous patcher.

Reply:
That wasn't a clean patch as the file had changed on those lines...

Reply:
Nevertheless, the changes are the same or similar enough. :)

Reply:
Changed.